docs/source/deep-learning/pytorch/index.rst

```diff
@@ -18,6 +18,30 @@ in MLflow we provide a set of APIs for:
 - **Experiments Management**: Store your PyTorch experiments in MLflow server, and you can view and share them from MLflow UI.
 - **Effortless Deployment**: Deploy PyTorch models with simple API calls, catering to a variety of production environments.
 
+5 Minute Quick Start with the MLflow PyTorch Flavor
+----------------------------------------------------
+
+To get a quick overview of how to use the MLflow PyTorch flavor, please read the quickstart guide. It
+will walk you through the basics of tracking PyTorch experiments.
+
+.. raw:: html
+
+    <a href="quickstart/pytorch_quickstart.html" class="download-btn">View the Quickstart</a>
+
+To download the PyTorch quickstart notebook to run in your environment, click the respective links below:
```

Comment:
Are we creating more than 1 quickstart? If not, we can singularize the reference here

Reply:
good call, fixed.